### ISSUE TITLE
Bug: Redis timeout on large multi op

### ISSUE THREAD

**Issue by mster** [mster/fireferret#1]
Large batch hash set crashes Redis client. 

```js
RedisError
    at RedisClient.multihset (/home/m/code/node/fireferret/lib/redis.js:151:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

**Comment by mster** [comment]
Based on ops/sec rate limit

**Comment by mster** [comment]
`OOM command not allowed when used memory > 'maxmemory'`

Has to do with cache fullness.

Using Volatile-LRU eviction policy results in the above OOM error. This is because keys are removed based off an the LRU algorithm's assigned timestamp.

Using Allkeys-LRU eviction policy resolves issues, as keys are removed when cache size grows.

**Comment by mster** [comment]
As eviction-policy is set configured outside of FireFerret's control, we may only make recommendations. 

A fallback method must be determined for this specific error.

**Comment by mster** [comment]
> A fallback method

Still required

**Comment by mster** [comment]
Bump

**Comment by mster** [comment]
Buckets reduce the amount of operations done by Redis.

SEE: #17

**Comment by mster** [comment]
Closed by #17
